examples/asr/emformer_rnnt/pipeline_demo.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import pathlib
+from argparse import ArgumentParser
+
+import torch
+import torchaudio
+from common import MODEL_TYPE_LIBRISPEECH, MODEL_TYPE_TEDLIUM3
+from torchaudio.pipelines import EMFORMER_RNNT_BASE_LIBRISPEECH
+from torchaudio.prototype.pipelines import EMFORMER_RNNT_BASE_TEDLIUM3
+
+
+logger = logging.getLogger()
+
+
+def get_dataset(model_type, dataset_path):
```

Comment:
Instead of repeating the key validation here and there, but these options in a dictionary, sthen pass the keys to `choices` arguments of argpsrse. So that the closed set of available options are defined once and only once.